pkg/gator/test/types.go

```diff
@@ -64,3 +64,25 @@ func (r *GatorResponses) Results() []*GatorResult {
 
 	return res
 }
+
+type testOptions struct {
+	referentialData bool
+}
+
+func defaultTestOptions() *testOptions {
+	return &testOptions{
+		referentialData: true,
+	}
+}
+
+func mutateTestOptions(ops *testOptions, mutators ...TestOptionMutator) {
```

Comment:
Could probably just call this `setTestOptions` to avoid confusion with mutation.